cypress/TestCases/FireboltCertification/Manage/AcknowledgeChallengeManage_RpcOnly.feature

```diff
@@ -0,0 +1,31 @@
+ Feature: AcknowledgeChallenge_Manage_RpcOnly
+
+    # Note: Reboot required after running this feature file
+
+    # Device manifest need to have the grantpolicy for this testcase to work
```

Comment:
cannot keep words like device manifest in FCS. That is platform specific and cannot be open sourced